### PR TITLE
PLANET-6083: Add UTM Parameters to share buttons

### PR DESCRIPTION
**Pull request by GP-Dan-Tovbein**
The Social Media share message should include the current page URL with the additional UTM parameters below (based on the [Global UTM Standards](https://app.gitbook.com/@greenpeace/s/insights/global-property/standards/utm-conventions))

Ref: https://jira.greenpeace.org/browse/PLANET-6083

32549_table

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [umbriel](https://www-dev.greenpeace.org/test-umbriel)

:watch: 2021.07.23 08:45:55

File: templates/blocks/share_buttons.twig [hunk 1]
```diff
@@ -1,27 +1,27 @@
 <div class="share-buttons">
 	<!-- Whatsapp -->
-	<a href="https://wa.me/?text={{ social.link }}"
+	<a href="https://wa.me/?text={{ social.link }}&utm_source=whatsapp&utm_medium={{ utm_medium }}{{ utm_campaign_param }}"
```
Comment (comzeradd) [inline templates/blocks/share_buttons.twig:3]: Shouldn't `utm_content` also be part of these urls?

Comment (GP-Dan-Tovbein) [reply templates/blocks/share_buttons.twig:3]: As far as I understand, we don't need to include it if the local campaign isn't set.

> The utm_campaign value should be based on the “Local Project” value defined in Planet 4 Analytics settings. All values should be in lowercase and spaces should be transformed to dashes . If no value is selected, we don’t need to include the utm_campaign value. 

Correct me if I'm wrong

Comment (GP-Dan-Tovbein) [reply templates/blocks/share_buttons.twig:3]: Good catch! I missed to add it.

Comment (GP-Dan-Tovbein) [reply templates/blocks/share_buttons.twig]: @comzeradd added

**Review by comzeradd — APPROVED** [review]
👍🏻

**Pull request merged**